src/main.py

```diff
@@ -127,8 +162,93 @@ async def on_member_update(self, prev, cur):
         if cur.activity.type is discord.ActivityType.playing:
             user.add_games([cur.activity.name])
             logging.info(
-                f'User starting playing {cur.activity.name}. Added to gamelist')
-
+                f'User starting playing {cur.activity.name}. Added to user gamelist')
+            self.add_games_to_gamelist(cur.activity.name)
+
+    def add_games_to_gamelist(self, games):
+        with open('GameList.json', 'r') as json_record:
```

Comment:
Should really use a database :P python has a pure python sqlite3 implementation that on startup we load the existing database (or create one if it does not exist). This could probably be raises as an "Issue" for one of us (I'd be happy to do it) us to implmeent